frontends/concrete-python/concrete/fhe/compilation/module.py

```diff
@@ -177,12 +207,12 @@ def encrypt(
             return tuple(args) if len(args) > 1 else args[0]  # type: ignore
         return self.execution_runtime.val.client.encrypt(*args, function_name=self.name)
 
-    def run(
+    def run_sync(
         self,
         *args: Optional[Union[Value, Tuple[Optional[Value], ...]]],
-    ) -> Union[Value, Tuple[Value, ...]]:
+    ) -> Any:
```

Comment:
Note if you have better way to force pyling to force return type to `Union[Value, Tuple[Value, ...]]`

Reply:
Hmm probably you have to do something in this spirit:
```
output = self._run(True, *args)
assert(isinstance(output, Union[Value, Tuple[Value, ...]])
```
And pylint will be happy after that ....

Reply:
Yeah, it's unfortunate that `typing.overload` doesn't support specifying overloads based on specific values.